lib/objectid.js

```diff
@@ -410,5 +410,5 @@ ObjectId.prototype.inspect = ObjectId.prototype.toString;
  */
 ObjectId.index = ~~(Math.random() * 0xffffff);
 
-Object.defineProperty(ObjectId.prototype, '_bsontype', { value: 'ObjectId' });
+Object.defineProperty(ObjectId.prototype, '_bsontype', { value: 'ObjectID' });
```

Comment:
Can you please leave a note here for future readers as to why we have reverted this?

Reply:
Yep, done.